ansto_simplon_api/schemas/ansto_endpoints.py

```diff
@@ -0,0 +1,9 @@
+from typing import Literal
+
+from pydantic import BaseModel, Field
+
+
+class LoadHDF5File(BaseModel):
+    hdf5_file_path: str = Field(examples=["/path/to/master_file"])
+    number_of_datafiles: int | None = Field(default=None, examples=[1])
+    compression: Literal["bslz4", "none"] = Field(default="bslz4", examples=["bslz4"])
```

Comment:
I guess technically, you could also have "bs" and "lz4" compression but I don't see when we would ever use that. 

Mind if we add if for future support?

Reply:
Yeah, I remember I tried to implement lz4, but I couldn't get it to work properly. It is related to the bytes_size_header that has to be added ([like this one](https://github.com/AustralianSynchrotron/ansto-simplon-api/blob/c6a80f31f3fc151b2ab3d91c198b9f14690415fd/ansto_simplon_api/simulate_zmq_stream.py#L233)). I'd have to look into it again and see what header works